CIScripts/Report/Repair-NUnitReport.ps1

```diff
@@ -16,6 +16,38 @@
     return $XML.OuterXml
 }
 
+function Split-NUnitReport {
+    Param([Parameter(Mandatory = $true)] [string] $InputData)
+    [xml] $XML = $InputData
+
+    $Root = Find-RootPesterSuiteNode -XML $XML
+
+    $PesterFilesNodes = $Root.Node.FirstChild.ChildNodes
+    $NumOfPesterFilesNodes = $PesterFilesNodes.Count
+
+    $XMLClones = 1..$NumOfPesterFilesNodes | ForEach-Object { $XML.Clone() }
+
+    $NodeToKeepIdx = 0
+    $XMLs = $XMLClones | ForEach-Object {
+        $RootOfClonedXML = Find-RootPesterSuiteNode -XML $_
+        $KeptNode = Copy-NodeOfSpecificTestSuite -FromNode $RootOfClonedXML
```

Comment:
`Copy-NodeOfSpecificTestSuite` uses `$NodeToKeepIdx` internally without passing it as a parameter using PowerShell magic.
Could you add a index parameter to this function?